### PR TITLE
"Wifi" -> "Wi-Fi"

### PR DESCRIPTION
**Pull request by GeneralFailer**

### CODE REVIEW

**Comment by Frenzie** [comment]
In regular written English *wifi*, *wi-fi*, *Wifi*, *WiFi* and *Wi-Fi* are all attested and acceptable. The fact that the latter is the official spelling of the Wi-Fi Alliance does not obligate us to any particular editorial practices besides internal consistency. I'd actually prefer to go with *wifi* or *wi-fi*, but of course the obvious alternative is *wireless network*.

File: frontend/ui/networkmgr.lua [hunk 1]
```diff
@@ -63,7 +63,7 @@ end
 
 function NetworkMgr:promptWifiOn()
     UIManager:show(ConfirmBox:new{
-        text = _("Do you want to turn on Wifi?"),
+        text = _("Do you want to turn on Wi-Fi?"),
```
Comment (Frenzie) [inline frontend/ui/networkmgr.lua]: Specifically
> Do you want to turn on the wireless network?

File: frontend/ui/networkmgr.lua [hunk 2]
```diff
@@ -72,7 +72,7 @@ end
 
 function NetworkMgr:promptWifiOff()
     UIManager:show(ConfirmBox:new{
-        text = _("Do you want to turn off Wifi?"),
+        text = _("Do you want to turn off Wi-Fi?"),
```
Comment (Frenzie) [inline frontend/ui/networkmgr.lua]: > Do you want to turn off the wireless network?

File: frontend/ui/networkmgr.lua [hunk 3]
```diff
@@ -97,7 +97,7 @@ end
 
 function NetworkMgr:getWifiMenuTable()
     return {
-        text = _("Wifi connection"),
+        text = _("Wi-Fi connection"),
```
Comment (Frenzie) [inline frontend/ui/networkmgr.lua]: This one doesn't could go either way. As a menu item the brusqueness of any spelling of *wifi* is probably desirable over something like *wireless connection* or *wireless network connection*.

**Comment by GeneralFailer** [comment]
I've misinterpreted your message at first glance and that played a big role in changing my mind when rereading it properly. On second thought Wi-Fi looks a bit better IMHO and isn't a bit confusing for people not used to informal written English.

**Comment by Frenzie** [comment]
I wasn't talking about informal written English.

**Comment by GeneralFailer** [comment]
Since _wifi_ or any other capitalization forms still hasn't made it into any major printed dictionaries I'd argue that it's technically mostly on the informal territory, although widely used.

**Comment by GeneralFailer** [comment]
Dammit, I keep confusing these buttons.

**Comment by houqp** [comment]
I just checked on my android phone and mac, they are all using `Wi-Fi`, so it's not bad to keep inline with them ;)

**Comment by Frenzie** [comment]
PS just laying out the argument. ;) The argument for Wifi is the same as
for e.g. Asus instead of ASUS.

**Comment by GeneralFailer** [comment]
Wi-Fi doesn't look bad for no good reason though and is probably often preferred in stuff like manuals of styles.

**Comment by houqp** [comment]
I actually like the idea of "wireless network". Ideally, all these protocol details should be abstracted away from user. The only thing they need to care about is whether the device is online or not and KOReader should take care of the rest. Hopefully we will get there one day :)

**Comment by Frenzie** [comment]
@GeneralFailer You could also say that Wi-Fi looks horrible for a horrible reason (promoting a trademark we have no interest in promoting, especially since we're just using it as a synonym for *wireless network*). \*grins\* ;) In any case, e.g. the PC World manual of style is for geeks. A quick peek on COCA shows that e.g. Mother Jones, The Christian Science Monitor, and the New Statesman all go for *wifi*. Admittedly, COCA does show an overwhelming preference for WiFi and Wi-Fi, without all of the fun variety you can find in the spelling and capitalization of hi-fi (HiFi, HI-FI, etc.).

Anyway, thanks for contributing. =D

**Pull request merged**